assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/rh-che.properties

```diff
@@ -22,3 +22,8 @@ che.fabric8.workspaces.routing_suffix=NULL
 che.fabric8.multitenant=false
 che.fabric8.user_service.endpoint=https://api.openshift.io/api/user/services
 
+# Enable or disable authentication using service account token retrieved using secrets
+che.openshift.service_account.auth_enable=false
```

Comment:
@garagatyi maybe it would be more handy not having `che.openshift.service_account.auth_enable` property (will need to open a new housekeeping issue for adding it to prod / prod-preview cm and update rh-che openshift templete) but rather have a fail-safe impl. of obtaining the token when it would be blank if the exception occurs ?

Reply:
Sorry, I didn't get last part of the suggested idea. On which values such an implementation would rely on?

Reply:
Well, I just do not understand the need of extra property, could we just do return [1] if other properties e.g. `che.openshift.service_account.id` etc are blank ?

[1] https://github.com/redhat-developer/rh-che/pull/538/files#diff-f2821ac62d8158e3bafa443f0876cddeR104

Reply:
Oh, I see. Yeah, we can do that. But I still need to check it on minishift to figure out what is the behavior when secret is missing in the namespace